### PR TITLE
add substatus to tracking status

### PR DESCRIPTION
**Pull request by dochoffiday**

### CODE REVIEW

**Review by jfriedr — APPROVED** [review]
Looks good.  This matches the format associated with our sub-status and adds a validation test 
image

**Comment by jfriedr** [comment]
@dochoffiday Thanks for the pull request.  While I'm not :100: it seems the reason I'm not able to merge this to master at the moment is due to a service outage w/ github webhooks (the circle CI tests have not yet passed as far as GH is concerned).
image

Once that's resolved (hopefully later today).  I'll let you know when this has been merged and published.

**Comment by dochoffiday** [comment]
thanks @jfriedr!

File: Shippo/Substatus.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+namespace Shippo
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class Substatus
+    {
+        [JsonProperty(PropertyName = "code")]
+        public string Code;
+
+        [JsonProperty(PropertyName = "text")]
+        public string Test;
```
Comment (jfriedr) [inline Shippo/Substatus.cs]: @dochoffiday I'm hoping that another git commit (or empty commit `git commit --allow-empty -m "Commit message here"`) will trigger the github / cirlceci exchange (to kick off the passing tests).  Would it be possible to amend this line to `public string Text;`

Comment (dochoffiday) [reply Shippo/Substatus.cs]: 👍 done

**Comment by dochoffiday** [comment]
@jfriedr any way to get this pushed through?

**Review by jfriedr — APPROVED** [review]

**Review by epistemancer — APPROVED** [review]
LGTM as well

**Pull request merged**